pkg/util/flags/flags.go

```diff
@@ -37,6 +37,22 @@ func VerifyRequiredFlags(cmd *cobra.Command, requiredFlags []string) error {
 	return createRequiredFlagError(cmd.Name(), missingFlags)
 }
 
+func VerifyRenamedFlags(cmd *cobra.Command, renamedFlags [][2]string) error {
```

Comment:
this function might be easier to read/maintain (especially where it is called and where it is tested) if it used a struct instead of a 2 element slice of strings.  For example `renamedFlags []RenamedFlag`, where the type `RenamedFlag` is defined as:
```
type RenamedFlag struct {
    NewFlagName string
    OldFlagName string
}
```

Reply:
Refactored under your suggestion